src/pages/storage/CustomVolumeSelectModal.tsx

```diff
@@ -115,7 +115,7 @@ const CustomVolumeSelectModal: FC<Props> = ({
       <NotificationRow />
       <ScrollableTable
         dependencies={[volumes, rows, notify.notification]}
-        belowId="modal-footer"
+        belowIds={["modal-footer", "status-bar"]}
```

Comment:
Here we don't have a status-bar in the modal.
```suggestion
        belowIds={["modal-footer"]}
```